premailer/premailer.py

```diff
@@ -420,6 +442,14 @@ def transform(self, pretty_print=True, **kwargs):
                 parent = item.getparent()
                 del parent.attrib['class']
 
+        # Capitalize Margin properties
+        # To fix weird outlook bug
+        # https://www.emailonacid.com/blog/article/email-development/outlook.com-does-support-margins
+        if self.capitalize_float_margin:
+            for item in page.xpath('//@style'):
```

Comment:
Could you be more specific on where to hook-in ? I don't remember where it was exactly in the code but the only places I saw when first looking for somewhere to plug my change were abstracted in a way that didn't let me specify the case of the css property.

Thanks :)